### PR TITLE
Helm updates

### PR DESCRIPTION
**Pull request by JeremyKennedy**
I've pulled all the changes from `next-template` and applied them to our project. Not sure if there's anything I've missed, and I don't think I can test this without merging it all in.

At first glance, does this look okay?

### CODE REVIEW

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -87,7 +114,178 @@ object Build: BuildType({
...
-})
\ No newline at end of file
+})
+
+object Build_Performance: BuildType({
+    name = "Build_Performance"
+    description = "Manually run performance environment"
+    params {
+        param("teamcity.vcsTrigger.runBuildInNewEmptyBranch", "true")
+        param("env.PROJECT", "next-template")
```
Comment (JeremyKennedy) [inline .teamcity/settings.kts]: should this `PROJECT` be set to `coabe`?

File: .teamcity/pom.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <project>
   <modelVersion>4.0.0</modelVersion>
-  <name>Dev_single-tier-appliction Config DSL Script</name>
-  <groupId>Dev_single-tier-appliction</groupId>
-  <artifactId>Dev_single-tier-appliction_dsl</artifactId>
+  <name>Dev_next-template Config DSL Script</name>
```
Comment (JeremyKennedy) [inline .teamcity/pom.xml]: should these names be updated to `coabe`?

File: .teamcity/settings.kts [hunk 3]
```diff
@@ -20,34 +22,59 @@ To debug in IntelliJ Idea, open the 'Maven Projects' tool window (View
...
+    buildType(CleanUpWeekly)
 }
 
-object HttpsGithubComDtsStnsingleTierAppliction : GitVcsRoot({
-    name = "https://github.com/DTS-STN/single-tier-appliction"
-    url = "git@github.com:DTS-STN/single-tier-appliction.git"
-    branch = "refs/heads/adding-teamcity-steps"
+object Dev_NextTemplate_HttpsGithubComDtsStnnextTemplateRelease : GitVcsRoot({
+    name = "https://github.com/DTS-STN/next-template/tree/_release"
+    url = "git@github.com:DTS-STN/next-template.git"
```
Comment (JeremyKennedy) [inline .teamcity/settings.kts:38]: `coabe`

Comment (JeremyKennedy) [inline .teamcity/settings.kts]: should these be updated to the `coabe` repo?

File: .teamcity/settings.kts [hunk 4]
```diff
@@ -20,34 +22,59 @@ To debug in IntelliJ Idea, open the 'Maven Projects' tool window (View
...
+/* Try and keep env.PROJECT value will be used throughout the helm scripts                 */
+/* to build urls, name the application and many other things.  folders and files in the    */
+/* helmfile directory should also match this value.                                        */
+object Build_Main: BuildType({
+    name = "Build_Main"
+    description = "Deploys main branch code on branch updates"
     params {
-        param("env.PROJECT", "single-tier-appliction")
+        param("teamcity.vcsTrigger.runBuildInNewEmptyBranch", "true")
+        param("env.PROJECT", "next-template")
```
Comment (JeremyKennedy) [inline .teamcity/settings.kts]: should this `PROJECT` be set to `coabe`?

File: .teamcity/settings.kts [hunk 5]
```diff
@@ -87,7 +114,178 @@ object Build: BuildType({
...
+        }
+    }
+})
+
+object CleanUpWeekly: BuildType({
+    name = "CleanUpWeekly"
+    description = "Deletes deployments every Sunday"
+    params {
+        param("teamcity.vcsTrigger.runBuildInNewEmptyBranch", "true")
+        param("env.PROJECT", "next-template")
```
Comment (JeremyKennedy) [inline .teamcity/settings.kts]: should this `PROJECT` be updated?

**Review by MarcoGoC — DISMISSED** [review]

**Comment by alex-solo** [comment]
Were these questions resolved? Ben would be a good resource

**Review by ben851 — COMMENTED** [review]
should the project name be updated? Yes, it should be changed from next-template to whatever your project is

**Comment by JeremyKennedy** [comment]
Note to myself and @MarcoGoC and anyone else reviewing this: we should compare with this to see what needs to be changed: https://github.com/DTS-STN/sc-digital-centre

**Comment by JeremyKennedy** [comment]
I think I will make the rename changes, and then merge it, and then we will have @MarcoGoC test the deployments.
Edit; I see Marco's made the changes already 😃

File: .teamcity/settings.kts (hunk 1 above)
Comment (MarcoGoC) [reply .teamcity/settings.kts]: changed

File: .teamcity/pom.xml (hunk 2 above)
Comment (MarcoGoC) [reply .teamcity/pom.xml]: changed

File: .teamcity/settings.kts (hunk 3 above)
Comment (MarcoGoC) [reply .teamcity/settings.kts]: changed

File: .teamcity/settings.kts (hunk 4 above)
Comment (MarcoGoC) [reply .teamcity/settings.kts]: changed

File: .teamcity/settings.kts (hunk 5 above)
Comment (MarcoGoC) [reply .teamcity/settings.kts]: changed

**Review by MarcoGoC — APPROVED** [review]
lets hope it works

**Pull request merged**